### PR TITLE
Feature/task01 html css

### PR DESCRIPTION
**Pull request by yecoAK**
##Description

The design came from the [One Piece Training Figma File](https://www.figma.com/file/QLbxC7r0B3GvNlDxJNEHKr/Training_ResponsiveWebsite_OnePiece).

Use the same images, font family, font size, and follow the distances of the designs.
Also, design the responsive page to Mobile view and table view

### CODE REVIEW

File: task01-html-css/root/index.html [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+                </a>
+
+                <nav class="nav">
+                    <ul class="menu" >
+                        <li><a href="#" class="menu active">Home</a></li>
+                        <li><a href="#">Character</a></li>
+                        <li><a href="#">Inslands</a></li>
+                        <li><a href="#">Mist Object</a></li>
+
+<!--                        <div class="ps-2 d-flex justify-content-end">-->
```
Comment (LeonelGodinez) [inline task01-html-css/root/index.html]: Why is this commented?

Comment (yecoAK) [reply task01-html-css/root/index.html]: left the comment by mistake, and I already deleted it

File: task01-html-css/root/css/styles.css [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+body {
+    font-family: 'Roboto', serif;
+}
+
+a {
+    color: #0C75AF;
+    text-decoration: none
+}
+a:hover{
+    text-decoration: underline !important;
```
Comment (LeonelGodinez) [inline task01-html-css/root/css/styles.css]: Why did you use !important here?

Comment (yecoAK) [reply task01-html-css/root/css/styles.css]: I have no idea, I forget to delete that, but I already deleted it

**Pull request merged**